src/checkbox-switch.ts

```diff
@@ -0,0 +1,12 @@
+import { window, workspace } from 'vscode';
+import { getKeywords, getUniq } from './utils';
+
+export default function (checkbox: string, action: string) {
+    if (checkbox.length != 3) {
+        window.showErrorMessage(`Text '${checkbox}' is not a checkbox`);
```

Comment:
The error message could say that only `[ ]` and `[X]` are considered valid checkboxes. The regexp below accepts other strings such as `[x]`, and the current error message does not tell user how to fix this unsupported input.

Note: org-mode also accepts `[-]` as indicating an "intermediate state", and users may try it. It's fine to not support the feature as long as users can easily discover (for example, in the error message) what parts are supported.